### ISSUE TITLE
New install on fresh Ubuntu - additions to readme

### ISSUE THREAD

**Issue by tiemes** [cybathlon-at-school/planvec#18]
My memos after a fresh install on Ubuntu. Could be used to improve the readme:

Installation

    Make sure git is installed. If not (e.g. on a new machine): sudo apt install git.
    Install zlib sudo apt-get install zlib1g-dev.
    Install pip sudo apt install python-pip.
		->sudo apt install python3-pip
    Clone this repo to a location of your choice.
		->cd irgendwo hin, zB. cd Desktop
		->git clone https://github.com/cybathlon-at-school/planvec.git
	
	Install pyenv. For instructions, see https://github.com/pyenv/pyenv.
		->autoinstaller (https://github.com/pyenv/pyenv-installer)
			prerequisites für ubuntu: (https://github.com/pyenv/pyenv/wiki/Common-build-problems)
				->sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
	
  Also run (from pyenv common build problems)

    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
    xz-utils tk-dev libffi-dev liblzma-dev python-openssl

	pyenv selber:
		curl https://pyenv.run | bash
		exec $SHELL
		
		(jenachdem: sudo nano ~/.bashrc
		# Load pyenv automatically by adding
# the following to ~/.bashrc:

export PATH="/home/toby/.pyenv/bin:$PATH"
eval "$(pyenv init -)"
eval "$(pyenv virtualenv-init -)"
)

		
    Install python 3.6.9 with pyenv 
	->pyenv install 3.6.9
	
    Install pipenv and do pipenv install (For instructions, see https://github.com/pypa/pipenv.) before pipenv shell inside planvec.
	->cd planvec
	->sudo apt install pipenv
	->pipenv shell
	
    
    Go to your cloned repo and do pipenv install and pipenv shell
	.
    Install a new ipykernel to be used inside a jupyter notebook:
    python -m ipykernel install --user --name planvec --display-name "planvec env"
	->Fehler, 
		pip install ipykernel
			->afterwards: python -m ipykernel install --user --name planvec --display-name "planvec env"

    Install pdfunite (contained in poppler-utils)
    sudo apt install install poppler-utils
		->error:  
		->pip install poppler-utils works
	Install pdfjam (contained in texlive-extra-utils) - takes a few minutes
    sudo apt-get install texlive-extra-utils

**Comment by matthaeusheer** [comment]
Nice. Eine saubere Lösung wäre sicherlich auch das ganze in einen Docker Container zu packen, je nachdem ob ihr jemanden habt, der damit Erfahrung hat. https://hub.docker.com/r/fadawar/docker-pyqt5 (habs nicht getested, aber könnte man wohl brauchen).

**Comment by tiemes** [comment]
Ja das wäre sehr toll. Ich persönlich kenne mich nur ganz minim damit aus, muss mal schauen, ob wir das irgendwie hinkriegen. Ich würde es wahnsinnig gerne auf einem Raspi laufen lassen.

**Comment by matthaeusheer** [comment]
Yes, das macht definitiv Sinn. Lass mich wissen, wie das läuft (ich glaube die processing pipeline ist jetzt nicht unbedingt auf speed oder wenig ressourcen optimiert :-) ).
